### ISSUE TITLE
(Self-Voicing general) Issue with visibility/hiding of Self-Voicing Quick Menu 

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#240]
![Screen Shot 2020-10-08 at 11 43 17 AM](https://user-images.githubusercontent.com/1902116/95470587-88feb580-095b-11eb-9f25-4e6641f0bb89.png)
@jessegreenberg, I just noticed that unchecking "Show Self Voicing Quick Menu" only hides the open/close button.
If I have the menu open before I hide it, the menu stays on the screen.

![Screen Shot 2020-10-08 at 11 40 05 AM](https://user-images.githubusercontent.com/1902116/95470171-0b3aaa00-095b-11eb-8a65-1a9edc54de25.png)

![Screen Shot 2020-10-08 at 11 41 49 AM](https://user-images.githubusercontent.com/1902116/95470435-5a80da80-095b-11eb-8992-2e8c75ae187e.png)

If the checkbox for show menu gets unchecked, it should hide everything whether or not it is opened or closed :-)

**Comment by terracoda** [comment]
And @jessegreenberg , I just noticed that the menu is not there if no content is checked.
![Screen Shot 2020-10-09 at 3 15 31 PM](https://user-images.githubusercontent.com/1902116/95617879-b07e7c80-0a46-11eb-8d87-ab01fedff365.png)

**Comment by terracoda** [comment]
@jessegreenberg, just pinging you in case you missed this issue. Sorry, I did not bring it up in meeting today.

**Comment by jessegreenberg** [comment]
Thanks @terracoda - in the above commit I made sure to close the menu when "Show Self Voicing Quick Menu" is unchecked.

> I just noticed that the menu is not there if no content is checked.

I think this was actually purposeful, I had thought that if self-voicing was disabled the menu should be invisible. But I can see it being valuable to still have the quick controls still work when object/context/hint responses are disabled.

In https://github.com/phetsims/scenery-phet/commit/1133f0f22bc63e1108dd64dfbe7ba367e61bc954 I made it so the buttons work if self-voicing checkboxes are unchecked.

@terracoda assigning to you to verify.

**Comment by jessegreenberg** [comment]
This has been fixed for a while, going to go ahead and close this.
